README.md

```diff
@@ -146,6 +146,11 @@ editor](https://github.com/mono/mono/blob/main/mcs/tools/csharp/getline.cs),
 dual-licensed under the terms of the MIT X11 license or the Apache License 2.0.
 Copyright (c) 2008 Novell, Inc. Copyright (c) 2016 Xamarin Inc.
 
+[src/stdlib/src/double-conversion](src/stdlib/src/double-conversion) includes
+content from the Google `double-conversion` library, available at
+https://github.com/google/double-conversion, licensed under the BSD 3-Clause
+"New" or "Revised" License. Copyright 2006-2011, the V8 project authors.
+
```

Comment:
We'll likely be able to drop this, in favor of the fmt lib: https://github.com/fmtlib/fmt. A few issues remains to be settled though, to make it "compatible enough" with `Double.ToString()` in .NET.